pyproject.toml

```diff
@@ -102,6 +102,7 @@ dependencies = [
   "requests",
   "protobuf==3.20",  # version minimum (for tests)
   "responses",
+  "aioresponses",
```

Comment:
`openai-python` uses `aiohttp` to make asynchronous requests, so we needed a new mock library